contracts/pcv/MigratorRouter.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {IMigratorRouter} from "./IMigratorRouter.sol";
+import {IPegStabilityModule} from "../peg/IPegStabilityModule.sol";
+import {IVolt} from "../volt/IVolt.sol";
+import {IVoltMigrator} from "../volt/IVoltMigrator.sol";
+
+/// @title Migrator Router
+/// @notice This contract is a router that wraps around the token migrator from
+/// the old volt ERC20 token to the new ERC20 token, to allow users to redeem for
+/// stables using the old volt version once the new version is live
+contract MigratorRouter is IMigratorRouter {
+    /// @notice the old VOLT token
+    IVolt public constant oldVolt =
+        IVolt(0x559eBC30b0E58a45Cc9fF573f77EF1e5eb1b3E18);
+
+    /// @notice VOLT-DAI PSM to swap between the two assets
+    IPegStabilityModule public immutable daiPSM;
+    /// @notice VOLT-USDC PSM to swap between the two assets
+    IPegStabilityModule public immutable usdcPSM;
+    /// @notice address of the new VOLT token
+    IVolt public immutable newVolt;
+    /// @notice the VOLT migrator contract to swap from old VOLT to new
+    IVoltMigrator public voltMigrator;
```

Comment:
there is no setter method to change this, so it can be made immutable and save on gas.